optaplanner-examples/src/test/java/org/optaplanner/examples/conferencescheduling/app/ConferenceSchedulingPerformanceTest.java

```diff
@@ -34,7 +34,7 @@ protected ConferenceSchedulingApp createCommonApp() {
     @Override
     protected Stream<TestData> testData() {
         return Stream.of(
-                testData(UNSOLVED_DATA_FILE, "-2115hard/0medium/-1350915soft", EnvironmentMode.REPRODUCIBLE),
-                testData(UNSOLVED_DATA_FILE, "-2700hard/0medium/-1379805soft", EnvironmentMode.FAST_ASSERT));
+                testData(UNSOLVED_DATA_FILE, "0hard/0medium/-1027755soft", EnvironmentMode.REPRODUCIBLE),
+                testData(UNSOLVED_DATA_FILE, "0hard/0medium/-1119825soft", EnvironmentMode.FAST_ASSERT));
```

Comment:
Does it get to this result in less than 5 or 10 seconds on your machine?
If it does, that's great, this test is safer now.
If it doesn't, this test is too slow, which is much worse than the fact it doesn't check if reaches a feasible score.

A performance test has a time budget of at most 10 seconds on a modern machine.
If it takes more than 10 seconds on your machine (a modern one), the test takes too long. For CI (which have much slower machines), a 10 second test on our machine can run for 5 minutes.
Long tests hurts developer productivity and CI validation speeds, so tests don't get more time budget than 10 machines on a modern machine which is 10 minutes (= timeout 600_000) on CI machines.

Reply:
I did that for all the tests. They all now finish within 10 seconds.